compiled_starters/javascript/app/main.js

```diff
@@ -3,6 +3,7 @@ const readline = require("readline");
 const rl = readline.createInterface({
   input: process.stdin,
   output: process.stdout,
+  terminal: false, // Disable input echoing
```

Comment:
My understanding is that if we want node/readline to function as terminal/shell, we'd leave the `terminal` option set to `true` by default. `terminal: true` is doing some extra stuff behind the scenes, such as "input echoing", which caused https://github.com/codecrafters-io/build-your-own-shell/issues/29

However, since our users are supposed to implement a shell, so we need node/readline to stay out of the way.